config/config.exs

```diff
@@ -31,3 +31,4 @@ import_config "#{Mix.env}.exs"
 
 config :pco_api, :api_key, System.get_env("PCO_API_KEY")
 config :pco_api, :api_secret, System.get_env("PCO_API_SECRET")
+# config :bypass, enable_debug_log: true
```

Comment:
I added it so it would be there when needed, but I guess setting it to false is probably better.